source/Plx.js

```diff
@@ -159,7 +159,7 @@ const FILTER_PROPERTIES = [
 ];
 
 // Props to be removed from passing directly to the component element
-const PROPS_TO_OMIT = [
+const PROPS_TO_OMIT = new Set([
```

Comment:
This is cool, but as code gets transpiled, it will be turned into array by babel anyway.

Reply:
Really? I did not inspect that. Is that because of the compatibility concern? If so, there should be a dynamic platform-specific code generation, I think, or we may sacrifice too much for IE, whose users will not expect so many animations I think.

Reply:
@Stanko Well according to [this answer](https://stackoverflow.com/questions/7700987/performance-of-key-lookup-in-javascript-object), I'm thinking we could have a better performance with just native object.